backend/tracim_backend/tests/functional/test_workspaces.py

```diff
@@ -4112,7 +4115,13 @@ def test_api__post_content_create_generic_content__ok_200__nominal_case(
         assert res.json_body["created"]
         assert res.json_body["file_extension"] == ".document.html"
         assert res.json_body["filename"] == "GenericCreatedContent.document.html"
-        params_active = {"parent_ids": 0, "show_archived": 0, "show_deleted": 0, "show_active": 1}
+        params_active = {
```

Comment:
Does this mean previous http request done by the frontend will failed or namespace_filter is optional ?

Reply:
namespaces_filter is optional (I didn't added it, it already existed probably for listing guest uploads).